versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    archive = {
+      source  = "hashicorp/archive"
+      version = "~> 1.3"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 2.70"
+    }
+  }
+  required_version = ">= 0.13"
```

Comment:
To extend the usage of the module can we accept since version 0.12?

Reply:
Definitely!